### DISCUSSION TITLE
Spock 2.4-M3

### DISCUSSION THREAD

**Discussion by leonard84** [spockframework/spock#1929] [Releases]
https://spockframework.org/spock/docs/2.4-M3/index.html

<hr /><em>This discussion was created from the release <a href='https://github.com/spockframework/spock/releases/tag/spock-2.4-M3'>Spock 2.4-M3</a>.</em>

**Comment by dlehammer** [comment]
Hi @leonard84 ,

I think the release link should point to https://spockframework.org/spock/docs/2.4-M3/release_notes.html ~ aligning with earlier releases 🤓
